NeuroRuler/CLI/main.py

```diff
@@ -0,0 +1,16 @@
+"""
+Defines main(), the entrypoint of the CLI (if you want to call it that).
+"""
+
+
+def main() -> None:
+    """Main entrypoint of CLI."""
+    pass
+
+
+if __name__ == "__main__":
+    import NeuroRuler.utils.parser as parser
+
+    parser.parse_config_json()  # TODO: are these all basically GUI settings? -Eric
```

Comment:
Yeah... I think we should make a separate JSON for actual program settings like rotation, slice, etc.

Reply:
I agree though I'm not sure how much of that should actually be specified in JSON or through arguments (though perhaps ideally anything should be overridable by args).